test/construct_ballot.test.ts

```diff
@@ -0,0 +1,153 @@
+import { AVClient } from '../lib/av_client';
```

Comment:
What are we actually testing?

Reply:
@lukelex, @av-emil and I have discussed the problem that you highlight here, and we all agree that it is wrong to use mocks from another test. Each test should have independant mocks. But for now we leave it as is. Any fixes to this should be addressed in a later pull request